package.json

```diff
@@ -46,6 +46,7 @@
     "grunt-wp-i18n": "~1.0.1",
     "istanbul": "^1.0.0-alpha",
     "mocha": "^3.0.2",
+    "serverpilot": "0.1.2",
```

Comment:
Could you share a little bit of why you decided to use serverpilot to create a WC instance to run the tests? I'm just curious to know a bit about your decision making process. I was under the impression that we would be able to configure a WC instance using just Travis but I'm probably wrong :) Or maybe it is possible but serverpilot is easier? Thanks.